src/Confluent.Kafka/AdminClientBuilder.cs

```diff
@@ -77,7 +77,7 @@ public AdminClientBuilder SetStatisticsHandler(Action<AdminClient, string> stati
         {
             if (this.StatisticsHandler != null)
             {
-                throw new ArgumentException("Statistics handler may not be specified more than once.");
+                throw new InvalidOperationException("Statistics handler may not be specified more than once.");
```

Comment:
this is not an argument exception because the argument is valid. `InvalidOperationException` much more appropriate here (and is a very commonly used exception).